coordinator/package.json

```diff
@@ -2,12 +2,12 @@
   "name": "electronic-prescription-service-api-coordinator",
   "version": "0.0.1",
   "description": "Electronic Prescription Service API Coordinator",
-  "main": "app.js",
+  "main": "coordinator/src/app.js",
   "scripts": {
-    "build": "tsc --alwaysStrict --project tsconfig-build.json",
+    "build": "tsc --build --clean && tsc --alwaysStrict --project tsconfig-build.json",
```

Comment:
tsc build works out the referenced project dependencies; builds these first, then outputs the transpiled javascript into the outDir side by side, in this case "dist" -> "coordinator/src", "models/library"